blocks/gdplus-subscribe-hero/gdplus-subscribe-hero.js

```diff
@@ -0,0 +1,149 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
+import { parseFragment, removeEmptyElements, render } from '../../scripts/scripts.js';
+
+const offerCards = [
+  {
+    title: 'Digital Only',
+    cost: '$1.99',
+    scope: 'mo',
+    billScope: 'annually',
+  },
+  {
+    title: 'Digital & Print',
+    cost: '$39.99',
+    scope: 'yr',
+    billScope: 'annually',
+    mostPopular: true,
+  },
+  {
+    title: 'Digital + Print + Golf Digest Schools',
+    cost: '$169.99',
+    scope: 'yr',
+    billScope: 'annually',
+  },
+];
+
+const benefits = [
+  {
+    title: 'Unlimited GolfDigest.com articles',
+  },
+  {
+    title: 'Digital subscription to Golf Digest magazine',
+  },
+  {
+    title: "Unlimited access to Golf Digest's 17,000+ course reviews",
+  },
+  {
+    title: 'Exclusive digital features, including interactive experiences',
+  },
+  {
+    title: 'Curated newsletters',
+  },
+  {
+    title: 'Print subscription to Golf Digest Schools',
+    info: 'Included only in the Digital + Print or Digital + Print + GD Schools plans',
+  },
+  {
+    title: 'Unlimited access to Golf Digest Schools',
+    info: 'Included only in the Digital + Print + Golf Digest Schools plans',
+  },
+  {
+    title: 'Unlimited access to the Golf Digest Archive - since 1950',
+  },
+];
+
+const HTML_TEMPLATE = `
+<div class="gd-plus-subscribe-content">
+  <div class="background-image-wrapper">
+    <div class="image-overlay"></div>
+    ${createOptimizedPicture('https://main--helix-sportsmagazine--headwirecom.hlx.live/subscribe-background-mockup.jpg', 'hero background green', true, [
```

Comment:
I don't think you need to put the origin `https://main--helix-sportsmagazine--headwirecom.hlx.live`.